app/javascript/components/history/helpers.js

```diff
@@ -1,41 +0,0 @@
-export function time_ago_in_words(date) {
```

Comment:
looks like `time_ago_in_words` is still being referenced in `_histories.html.erb`, what's the reasoning for unrefactoring this function from a separate file?

Reply:
We started putting all of our DateTime util functions in a separate file, so it was moved there.  I can update the reference

Reply:
looks like this file isn't even being used anymore, so I went ahead and removed it here: https://github.com/SaraAlert/SaraAlert/pull/793/commits/ad9df046c5344445c1cf056f8dfb316aee18461a